SevereServiceError not included in TypeScript types

[//]: # NOTE: This repository only maintains packages that are listed in the [Readme](https://github.com/webdriverio/webdriverio/blob/master/README.md#packages). Please make sure that your issue is directly caused by one of these packages and if not file an issue in the correct 3rd party package repository.

**Environment (please complete the following information):**
 - **WebdriverIO version:** 6.3.4
 - **Mode:** N/A - TypeScript compile error
 - **If WDIO Testrunner, running sync/async:** N/A - TypeScript compile error
 - **Node.js version:** 10.20.1
 - **NPM version:** npm 6.14.4 and yarn 1.22.0
 - **Browser name and version:** N/A
 - **Platform name and version:** Mac 10.14.6
 - **Additional wdio packages used (if applicable):** N/A

**Config of WebdriverIO**
N/A - TypeScript compile error.

**Describe the bug**
In PR #5565, the new SevereServiceError was added. The TypeScripts types were not updated.

**To Reproduce**
`mkdir severeserviceerror-types`
`cd severeserviceerror-types`
`npm init` (then accept all the defaults)
`npm add webdriverio`
`echo "import { remote, SevereServiceError } from 'webdriverio';" > test.ts`
`tsc test.ts` (assumes you have a global install of TypeScript)

**Expected behavior**

```Typescript
// test.ts
import { remote, SevereServiceError } from 'webdriverio';
```
This file should compile without errors.

**Log**

```
$ tsc test.ts
test.ts:1:18 - error TS2305: Module '"webdriverio"' has no exported member 'SevereServiceError'.

1 import { remote, SevereServiceError } from 'webdriverio';
                   ~~~~~~~~~~~~~~~~~~


Found 1 error.
```

**Additional context**
I have a local fix, I'll open a PR.

--- a/packages/webdriverio/src/utils/SevereServiceError.js
+++ b/packages/webdriverio/src/utils/SevereServiceError.js
@@ -1,8 +1,8 @@
 /**
- * Error to be thrown when a sever error was encountered when a Service is being ran.
+ * Error to be thrown when a severe error was encountered when a Service is being ran.
  */
 export default class SevereServiceError extends Error {
-    constructor(message = 'Sever Service Error occurred.') {
+    constructor(message = 'Severe Service Error occurred.') {
         super(message)
         this.name = 'SevereServiceError'
     }

--- a/packages/webdriverio/tests/index.test.js
+++ b/packages/webdriverio/tests/index.test.js
@@ -1,19 +1,26 @@
 import * as webdriverio from '../src'
 
+// If you're making a change here, like adding a new export, the TypeScript
+// typings may need an update : packages/webdriverio/webdriverio.d.ts
+
 describe('index.js', () => {
-    it('exports attach method', () => {
-        expect(webdriverio.attach).toBeDefined()
-    })
-
-    it('exports attach method', () => {
-        expect(webdriverio.multiremote).toBeDefined()
-    })
-
-    it('exports attach method', () => {
+    it('exports remote method', () => {
         expect(webdriverio.remote).toBeDefined()
     })
 
     it('exports attach method', () => {
+        expect(webdriverio.attach).toBeDefined()
+    })
+
+    it('exports multiremote method', () => {
+        expect(webdriverio.multiremote).toBeDefined()
+    })
+
+    it('exports remote method', () => {
+        expect(webdriverio.remote).toBeDefined()
+    })
+
+    it('exports SevereServiceError class', () => {
         expect(webdriverio.SevereServiceError).toBeDefined()
     })
 })

--- a/packages/webdriverio/tests/utils/SevereServiceError.test.js
+++ b/packages/webdriverio/tests/utils/SevereServiceError.test.js
@@ -8,7 +8,7 @@ describe('SevereServiceError', () => {
 
     it('should provide a default error message', () => {
         const err = new SevereServiceError()
-        expect(err.message).toBe('Sever Service Error occurred.')
+        expect(err.message).toBe('Severe Service Error occurred.')
     })
 
     it('should accept a custom error message', () => {

--- a/packages/webdriverio/webdriverio.d.ts
+++ b/packages/webdriverio/webdriverio.d.ts
@@ -40,6 +40,11 @@ declare namespace WebdriverIO {
     }
 
     interface BrowserObject extends WebDriver.ClientOptions, WebDriver.ClientAsync, Browser { }
+
+    /**
+     * Error to be thrown when a severe error was encountered when a Service is being ran.
+     */
+    class SevereServiceError extends Error { }
 }
 
 declare var browser: WebdriverIO.BrowserObject;

--- a/tests/typings/webdriverio/async.ts
+++ b/tests/typings/webdriverio/async.ts
@@ -1,7 +1,7 @@
 import allure from '@wdio/allure-reporter'
 import { remote, multiremote, MockOverwriteFunction } from 'webdriverio'
 
-// An example of adding command withing ts file to WebdriverIO (async)
+// An example of adding command within ts file to WebdriverIO (async)
 declare module "webdriverio" {
     interface Browser {
         browserCustomCommand: (arg: unknown) => Promise<void>

--- a/tests/typings/webdriverio/async.ts
+++ b/tests/typings/webdriverio/async.ts
@@ -1,5 +1,5 @@
 import allure from '@wdio/allure-reporter'
-import { remote, multiremote, MockOverwriteFunction } from 'webdriverio'
+import { remote, multiremote, MockOverwriteFunction, SevereServiceError } from 'webdriverio'
 
 // An example of adding command within ts file to WebdriverIO (async)
 declare module "webdriverio" {
@@ -249,6 +249,14 @@ async function bar() {
     const match = mock.calls[0]
     match.body
     match.headers
+}
+
+function testSevereServiceError_noParameters() {
+    throw new SevereServiceError();
+}
+
+function testSevereServiceError_stringParameter() {
+    throw new SevereServiceError("Something happened.");
 }
 
 // allure-reporter

--- a/tests/typings/webdriverio/types/async.d.ts
+++ b/tests/typings/webdriverio/types/async.d.ts
@@ -1,5 +1,5 @@
 /**
- * An example of adding command withing d.ts file to WebdriverIO (async)
+ * An example of adding command within d.ts file to WebdriverIO (async)
  */
 
 // module should be "webdriverio" if used within `ts` file instead of `d.ts`

